### DISCUSSION TITLE
pagination - keep last paginated images in focus

### DISCUSSION THREAD

**Discussion by IkramMaalej** [kitodo/kitodo-production#4727] [Q&A] [answered]
Concerning issue #4562

After pagination, the last paginated pages must remain selected in order not to scroll down to paginate further pages.
For this point there is another opinion that if the recently paginated pages are still selected, they must first be deselected in order to continue pagination.
Wich opinion is more reasonable?

**Comment by andre-hohmann** [comment, marked as answer]
Probably, this is a misunderstanding of my description in the issue. 

- I do not want to keep the selection-box selected, after the pagination of one or several images was conducted. 
- I want that the same images are shown after the pagination is conducted and not always image 1. If you have several different pagination forms at the end of the images, scrolling always down from the beginning becomes very annoying. 

In #4562 the current steps (at least in our version) are described: 
1. search and select Image 36 
2. assign "uncounted" to image 36
3. _after assigning the value, image 1 is shown_
4. _search and scroll for Image 37_ 
5. select Image 37 
6. assign "1" to image 37
7. _after assigning the value, image 1 is shown_
8. _search and scroll for Image 40_
9. select Image 40
10. assign "uncounted" to image 40
11. _after assigning the value, image 1 is shown_
12. _search and scroll for Image 41_
13. select Image 41
14. assign "5" to image 41
15. ...

In my opinion, the step "after assigning the value, image 1 is shown" and the search/scroll from image 1 in next step is not necessary. 

Here the way i suggest: 
1. search and select Image 36 
2. assign "uncounted" to image 36
3. _after assigning the value, image 36 is not selected, but shown_
4. select Image 37 
5. assign "1" to image 37
6. _after assigning the value,, image 37 is not selected, but shown_
7. select Image 40
8. assign "uncounted" to image 40
9. _after assigning the value, image 40 is not selected, but shown_
10. select Image 41
11. assign "5" to image 41
12. ...

This behaviour may not affect "normal" pagination. Maybe it is better to try the following pagination forms: 
- http://digital.slub-dresden.de/id358576229
- http://digital.slub-dresden.de/id368157962

Maybe it is helpful to see it - sorry for the bad quality, but otherwise i cannot upload the file: 
![PaginierungScrollen2](https://user-images.githubusercontent.com/14835770/136543152-c145a526-458c-4e22-b91a-c6a4593c0f66.gif)

**Reply by solth** [reply]
Thank you very much for the detailed description, @andre-hohmann ! So the real issue is that the page list in the pagination panel always scrolls to the top after performing any kind of pagination, and does not have anything to do with the page _selection_, right?

**Reply by andre-hohmann** [reply]
Exactly, i just do not want to scroll after performing any kind of pagination from image 1 to the needed one. 
If you conduct a complex pagination in the area of image 500, ... you will quickly understand why. 

**The page selection (or de-selection) after performing a pagination is absolutely fine as it is!**
